cloudformation/slack-router-lambda.js

```diff
@@ -0,0 +1,104 @@
+const AWS = require('aws-sdk')
+const crypto = require('crypto')
+
+const region = process.env.AWS_REGION
+const accountId = process.env.AWS_ACCOUNT_ID
+
+let ssmConnection
+function getSSMConnection() {
+  if (ssmConnection) return ssmConnection
+
+  ssmConnection = new AWS.SSM()
+  return ssmConnection
+}
+
+let snsConnection
+function getSNSConnection() {
+  if (snsConnection) return ssmConnection
```

Comment:
shouldn't this be `return snsConnection`?

Reply:
Absolutely right. Thanks for flagging!